1-WebApp-OIDC/1-2-AnyOrg/README.md

```diff
@@ -248,7 +248,7 @@ An example of code for `ValidateSpecificIssuers` is the following:
     {
         var validIssuers = GetAcceptedTenantIds()
                              .Select(tid => $"https://login.microsoftonline.com/{tid}");
-        if (validIssuers.Contains(issuer))
+        if (validIssuers.Any(vi => issuer.StartsWith(vi)))
```

Comment:
What problem is it solving, @ravimpatel ?
Is it that the issuer ends with v2.0 ?

Reply:
May be due to versioning changes, the issuer url returned looks like this https://login.microsoftonline.com/{tid}/v2.0 instead of https://login.microsoftonline.com/{tid}. This fails the Contains check and hence the issue.

Reply:
Thanks for the update, @ravimpatel 
I'll check something that works in all cases

Reply:
@jmprieur , looking forward to it. thanks. This is my first PR, so not familiar with the process. What should I do with it? Close it?